src/harness/testcases/spectrum_inquiry_testcase.py

```diff
@@ -391,6 +391,95 @@ def test_WINNF_FT_S_SIQ_16(self):
       self.assertEqual(resp['response']['responseCode'], 0)
 
   @winnforum_testcase
+  def test_WINNF_FT_S_SIQ_17(self):
+    """This test ensures that when a message with two requests, 
+    both for PAL channels, is sent with valid parameters, 
+    the SAS sends a successful response for both the requests.
+    Response Code must be 0 for all CBSDs
+    """
+
+    # Load the Data
+    pal_low_frequency = 3550000000.0
+    pal_high_frequency = 3560000000.0
```

Comment:
according to the new decision in the spec of SAS-CBSD the frequncy will be integer, but maybe we can do this modification later when we will change it everywhere on the master branch.